src/kagglehub/datasets.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from typing import Optional
+
+from kagglehub import registry
+from kagglehub.datasets_helpers import create_dataset_or_version
+from kagglehub.gcs_upload import upload_files_and_directories
+from kagglehub.handle import parse_dataset_handle
+from kagglehub.logger import EXTRA_CONSOLE_BLOCK
+
+logger = logging.getLogger(__name__)
+
+
+def dataset_download(handle: str, path: Optional[str] = None, *, force_download: Optional[bool] = False) -> str:
+    """Download dataset files
+    Args:
+        handle: (string) the dataset handle
+        path: (string) Optional path to a file within the dataset
+        force_download: (bool) Optional flag to force download a dataset, even if it's cached.
+    Returns:
+        A string requesting the path to the requested dataset files.
+    """
+
+    h = parse_dataset_handle(handle)
+    logger.info(f"Downloading Datset: {h.to_url()} ...", extra={**EXTRA_CONSOLE_BLOCK})
```

Comment:
let's remove this log statement and let the each resolvers handle the logging. For instance, if you are in a Kaggle environment, we will "attach" the dataset and not "download" it.

Reply:
Sorry, looks like Nesh added it for models and added an extra to exclude it from the console log (only logs to file). We can keep it to match models.

Make sure to fix the typo: "s/Datsaet/Dataset"